frontend/src/components/common/SectionFilterHeader.tsx

```diff
@@ -129,6 +130,9 @@ export default function SectionFilterHeader(props: SectionFilterHeaderProps) {
               <NamespacesAutocomplete />
             </Grid>
           )}
+          <Grid item>
+            <CRDCategoriesList />
+          </Grid>
```

Comment:
I don't think this should be part of the filter header, as this concerns the CRD list only.

Reply:
This has been removed from SectionFilterHeader.